src/_Form.sass

```diff
@@ -25,6 +25,9 @@ select
 		border: .1rem solid $color-primary
 		outline: 0
 
+select
+	appearance: menulist
```

Comment:
This way each browser renders your way. I plan to do another way. It has to have the same behavior in all browsers.